Submitted By: Zack Winkles <winkie@linuxfromscratch.org>
Date: 2003-10-05
Initial Package Version: 2.95.3
Origin: LFS-Dev archives
Description: Fixes gcc 2.95.3 so the -Wreturn-type flag will work, even when
g++ is not compiled.


diff -Naur gcc-2.95.3.orig/gcc/toplev.c gcc-2.95.3/gcc/toplev.c
--- gcc-2.95.3.orig/gcc/toplev.c	2003-05-28 16:09:29.000000000 -0400
+++ gcc-2.95.3/gcc/toplev.c	2003-05-28 16:16:05.425858120 -0400
@@ -1079,6 +1079,8 @@
   { "-Wno-nested-externs", "" },
   { "-Wparentheses", "Warn about possible missing parentheses" },
   { "-Wno-parentheses", "" },
+  { "-Wreturn-type", "Warn about inconsistent return types" },
+  { "-Wno-return-type", "" },
   { "-Wpointer-arith", "Warn about function pointer arithmetic" },
   { "-Wno-pointer-arith", "" },
   { "-Wredundant-decls",
